docs/content/formatting.mdx

```diff
@@ -0,0 +1,86 @@
+---
+name: Formatting
+menu: Content
+route: /content/formatting
+---
+
+# Formatting
+
+## Guidelines
```

Comment:
Should there be a general guidance here to follow user preferences when applicable?

Reply:
Yes indeed